setup.py

```diff
@@ -1769,16 +1769,15 @@ def detect_tkinter_darwin(self):
         for F in framework_dirs:
             # both Tcl.framework and Tk.framework should be present
 
-
             for fw in 'Tcl', 'Tk':
                 if is_macosx_sdk_path(F):
-                    if not exists(join(sysroot, F[1:], fw + '.framework')):
```

Comment:
Currently, this condition would match, because in macOS 10.15 `/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/Tk.framework` exists, however, `F` is not the path that it's checking, so it adds `/System/Library/Frameworks/Tk.framework` to the list of include_dirs